collectors/pool_usage.go

```diff
@@ -50,31 +54,44 @@ func NewPoolUsageCollector(conn Conn) *PoolUsageCollector {
 		UsedBytes: prometheus.NewGaugeVec(
 			prometheus.GaugeOpts{
 				Namespace: cephNamespace,
-				Name:      "pool_used_bytes",
+				Subsystem: "pool",
+				Name:      "used_bytes",
 				Help:      "Capacity of the pool that is currently under use",
 			},
 			[]string{"pool"},
 		),
+		MaxAvail: prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: cephNamespace,
+				Subsystem: "pool",
```

Comment:
Let's define a constant for the `pool` subsystem so that any future changes we decide to make, if necessary, would be done in one place.